app/controllers/file_controller.rb

```diff
@@ -50,7 +50,9 @@ def show
     disposition = params[:disposition] || 'inline'
     filename = params["filename"] || "#{objid}#{fileid}"
     headers['Content-Disposition'] = "#{disposition}; filename=#{filename}"
-    if ds.mimeType
+    if filename.include?('.jpg')
```

Comment:
@mcritchlow Both Ron and I checked that the 2.jpg files itself don't contain header information such as MIME type, do they?
For http protocol, the header information is usually added in web application, that's why I used option 1 to override the MIME metadata from triplestore.
But since we decided to try the option 2:
If the derivative is fine that are not corrupted(like the example in ticket), re-creating derivative by image-magic would't fix the problem, but we should try to re-running JHOVE to fix the metadata in triplestore.  
Otherwise, both need to re-generate. 
But the side effect is that is there possible the derivative was fine but after re-generate it got corrupted? If it happened before, it could happen again.